### PR TITLE
Release 1.27.2

### PR DESCRIPTION
**Pull request by dulek**
**What this PR does / why we need it**:
This releases 1.27.2 bugfix release.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: charts/openstack-cloud-controller-manager/Chart.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: v1.27.1
+appVersion: v1.27.2
 description: Openstack Cloud Controller Manager Helm Chart
 icon: https://object-storage-ca-ymq-1.vexxhost.net/swift/v1/6e4619c416ff4bd19e1c087f27a43eea/www-images-prod/openstack-logo/OpenStack-Logo-Vertical.png
 home: https://github.com/kubernetes/cloud-provider-openstack
 name: openstack-cloud-controller-manager
-version: 2.27.2
+version: 2.27.3
```
Comment (dulek) [inline charts/openstack-cloud-controller-manager/Chart.yaml:7]: What should I do with this now? @zetaab?

Comment (zetaab) [reply charts/openstack-cloud-controller-manager/Chart.yaml:7]: always increasing the version is the way to go

File: go.mod [hunk 2]
```diff
@@ -36,7 +36,7 @@ require (
 	k8s.io/component-base v0.27.1
```
Comment (zetaab) [inline go.mod:36]: we should perhaps update the other k8s.io 0.27.1 deps to 0.27.5? otherwise PR looks good

**Review by zetaab — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (dulek) [reply go.mod]: Done, please see if I did it correctly.

**Review by zetaab — APPROVED** [review]
/approve

**Comment by zetaab** [comment]
hmm, just thinking why cinder test is failing. It works in master + release-1.28 currently

**Comment by jichenjc** [comment]
/test openstack-cloud-csi-cinder-e2e-test-release-127

1.27 used to be working I think ?

**Comment by zetaab** [comment]
yeah it was working in https://github.com/kubernetes/cloud-provider-openstack/pull/2311

**Comment by dulek** [comment]
Let's see https://github.com/kubernetes/cloud-provider-openstack/pull/2349 then.

**Comment by dulek** [comment]
@zetaab: I just had a moment to take a closer look at the job failure. Looks like the 1.27.2 image is not available and that's why it fails:

```
Error from server (BadRequest): container "openstack-cloud-controller-manager" in pod "openstack-cloud-controller-manager-b9575fdc6-js89h" is waiting to start: trying and failing to pull image
```

Could it be because I cannot tag this as 1.27.2 and the images are not in staging? What's the procedure there?

File: tests/playbooks/roles/install-cpo-occm/defaults/main.yaml [hunk 3]
```diff
@@ -9,4 +9,4 @@ run_e2e: false
 # Used for access the private registry image from k8s
 remote_registry_host: "{{ ansible_default_ipv4.address }}"
 generated_image_url: "{{ remote_registry_host }}/openstack-cloud-controller-manager:v0.0.{{ github_pr }}"
-image_url: "{{ generated_image_url if build_image else 'registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.27.1' }}"
+image_url: "{{ generated_image_url if build_image else 'registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.27.2' }}"
```
Comment (zetaab) [inline tests/playbooks/roles/install-cpo-occm/defaults/main.yaml:12]: yeah the problem is coming from this row, the cinder e2e test does not build the image - instead tries to use image that does not exist yet. It is kind of chicken - egg problem.

Comment (zetaab) [reply tests/playbooks/roles/install-cpo-occm/defaults/main.yaml:12]: https://github.com/kubernetes/cloud-provider-openstack/pull/2351 this tries to fix this issue.

**Comment by dulek** [comment]
I've rebased this to include #2352, let's see now.

**Comment by dulek** [comment]
It went through!

**Pull request merged**